block/stream/output/schedule_service.proto

```diff
@@ -64,6 +64,17 @@ import "basic_types.proto";
  * the original transaction.
  */
 message CreateScheduleOutput {
+    /**
+     * A schedule identifier.
+     * <p>
+     * If the status of the transaction is `SUCCESS`, this value SHALL be
+     * set to the identifier of the schedule that was created.<br/>
+     * If the transaction status is `IDENTICAL_SCHEDULE_ALREADY_CREATED`,
+     * this value SHALL be set to the identifier of an existing schedule
+     * that is identical to the one that was requested.<br/>
+     * For any other status, this value SHALL NOT be set.
+     */
+    proto.ScheduleID schedule_id = 1;
```

Comment:
Should this be renamed to `duplicate_schedule_id`?
Also, if we only set this for duplicate schedules, we may need to remove the first specification sentence in the document comment above.